index.html

```diff
@@ -9,42 +9,61 @@
   </head>
 
   <body>
-    <nav class="nav">
-      <a href="/" class="nav__link" data-link>List</a>
-      <a href="/" class="nav__link" data-link>Add New</a>
-      <a href="/" class="nav__link" data-link>Contact</a>
-    </nav>
-    <div class="books" id="books">
-      <h1>Awesome Books</h1>
-      <table>
-        <tr>
-          <th>Title</th>
-          <th>Author</th>
-          <th></th>
-        </tr>
-        <tbody id="lists"></tbody>
-      </table>
+    <div class="navigation">
+      <h1 class="logo"><a href="">Awesome Books</a></h1>
+      <nav class="nav">
+        <a href="#" class="nav__link list" >List</a>
+        <a href="#" class="nav__link addbooks" >Add New</a>
+        <a href="#" class="nav__link contacts" >Contact</a>
 
-       <h1>Add a New Book</h1>
-      <form id="form" class="form">
-        <input
-          type="text"
-          name="author"
-          id="atr"
-          placeholder="author"
-          class="atr"
-        />
-        <input
-          type="text"
-          name="title"
-          id="ttl"
-          placeholder="title"
-          class="ttl"
-        />
-        <button id="btn">Add</button>
-      </form>
+      </nav>
     </div>
+
+    <main class="main">
+      <div class="books" id="books">
+        <div class="wrapper">
+          <h1>Awesome Books</h1>
+          <table class="lists">
+            <tbody id="lists"></tbody>
```

Comment:
- [ ] OPTIONAL
The styling of your _books list rows_ background color does not conform to the database table alternate rows color, as can be seen in the original wireframe below. Kindly ensure to improve upon this aspect.

Your design:
![image](https://user-images.githubusercontent.com/12745474/183250002-e6a168eb-d9dc-4023-a9db-1442fd17ae8d.png)

**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/134323111-24a1a54d-374a-4197-bdc5-7ddbbcc89636.png)